news/_posts/core-weekly/2019-06-17-coreweekly-week-24-2019.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 24 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-06-17 17:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 10th to Sunday 16th of June 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+While the last bugs and regressions are being fixed on the core in order to build a release candidate, you will notice that a lot of work has been done on modules too! And new versions of very popular native modules are coming :-)
+
+Stay tune for the release candidate.
+
+And of course, tomorrow, don't forget to join the PrestaShop team and partners at [the Madrid event](https://www.prestashop.com/es/eventos/prestashop-day-madrid).
+
+
+## A quick update about PrestaShop's GitHub issues and pull requests:
+
+- [65 new issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+created%3A2019-06-10..2019-06-16) have been created in the project repositories;
+- [61 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2019-06-10..2019-06-16), including [18 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2019-06-10..2019-06-16) on the core;
+- [81 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2019-06-10..2019-06-16) in the project repositories;
+- [62 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2019-06-10..2019-06-16), including [52 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2019-06-10..2019-06-16).
+
+## Code changes in the 'develop' branch
+
+### Core
+
+* [#14130](https://github.com/PrestaShop/PrestaShop/pull/14130): Merge 1.7.6.x to develop - 10/06/2019 [travis investigation - dont merge/close], by [@matks](https://github.com/matks)
```

Comment:
It would have been nice to remove it I think 😅

Reply:
why ?
then if someone count the PR on the core-weekly, the number will be different than in the resume, where the number of merged PR is available with a link.